src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,18 @@
 package core.basesyntax;
 
 public class MainApp {
+    private Machine[] machines = new Machine[]{new Truck(), new Bulldozer(), new Excavator()};
 
+    public Machine[] getMachines() {
+        for (Machine machine: machines) {
+            machine.doWork();
+            machine.stopWork();
+        }
+        return machines;
+    }
```

Comment:
what is this method for? It's named 'getMachines' but actually it performs doWork and stopWork operations on the list of machines. And the result of this method is just ignored. I'd recommend removing this method at all